tests/helpers/properties/rfc268-adapter.js

```diff
@@ -0,0 +1,37 @@
+
+export { test as testForAcceptance } from 'qunit';
+
+import $ from 'jquery';
+import { settled } from '@ember/test-helpers';
+
+export default function Rfc268Adapter() {}
+
+Rfc268Adapter.prototype = {
+  $(selector, isAlternative) {
+    return $(selector, isAlternative ? '#alternate-ember-testing' : '#ember-testing');
+  },
+
+  async throws(assert, block, expected, message) {
```

Comment:
not sure if it's the same, but can we use [`assert.rejects`](https://api.qunitjs.com/assert/rejects) here? At least currently we have the latest `qunit` installed which provides us with this api.

Reply:
I don't think it's quite the same...mumble mumble test adapter something something...I can't exactly recall but I think we have to do it just so to keep the test setup working correctly.